portfolio/style.css

```diff
@@ -0,0 +1,504 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+h1, h2, h3, h4, h5, h6, h7, button, a {
+  font-family: 'Noto Sans', sans-serif;
+}
+
+p, li {
+  font-family: 'Nunito', sans-serif;
```

Comment:
Whichever font you want to be the standard font for your page, you might want to put in a `body {}` class declaration.